src/frontend/js/widgets/Dashboard/components/DashboardItem/stories.mock.ts

```diff
@@ -17,6 +17,7 @@ export const enrollment: Enrollment = {
     resource_link: 'https://lms.fun-mooc.fr/courses/course-v1:supagro+120001+archive_ouvert/info',
     state: CourseStateFactory.generate(),
     course: {
+      id: '0001',
```

Comment:
Just to stick to the reality
```suggestion
      id: '1cc49d2b-fc08-46d2-9fb8-594d90494cd3',
```